src/components/shared/MarkerTooltipContents.js

```diff
@@ -308,41 +334,82 @@ function getMarkerDetails(
+                        'Cells tenured',
+                        formatValueTotal(
+                          nursery.cells_tenured,
+                          nursery.cells_tenured +
+                            nursery.cells_allocated_nursery,
+                          formatSI
+                        )
+                      )
+                    : null}
+                  {nursery.cur_capacity
+                    ? _markerDetail(
                         'gcnurseryusage',
                         'Bytes used',
                         formatValueTotal(
                           nursery.bytes_used,
                           nursery.cur_capacity,
                           formatBytes
                         )
-                      )}
-                  {nursery.new_capacity === undefined
-                    ? null
-                    : _markerDetail(
+                      )
+                    : null}
+                  {nursery.new_capacity
+                    ? _markerDetail(
                         'gcnewnurserysize',
                         'New nursery size',
                         nursery.new_capacity,
                         formatBytes
-                      )}
-                  {nursery.lazy_capacity === undefined
-                    ? null
-                    : _markerDetail(
+                      )
+                    : null}
+                  {nursery.lazy_capacity
+                    ? _markerDetail(
                         'gclazynurserysize',
                         'Lazy-allocated size',
                         nursery.lazy_capacity,
                         formatBytes
-                      )}
-                  {nursery.chunk_alloc_us === undefined
-                    ? null
-                    : _markerDetail(
+                      )
+                    : null}
+                  {nursery.cells_allocated_nursery &&
+                  nursery.cells_allocated_tenured
+                    ? _markerDetail(
+                        'gcnursaryallocations',
+                        'Nursery allocations since last minor GC',
+                        formatValueTotal(
+                          nursery.cells_allocated_nursery,
+                          nursery.cells_allocated_nursery +
+                            nursery.cells_allocated_tenured,
+                          formatSI
+                        )
+                      )
+                    : null}
+                  {evictTime
+                    ? _markerDetail(
+                        'gctenurerate',
+                        'Tenure rate (bytes/sec)',
+                        nursery.bytes_tenured / (evictTime / 1000000),
+                        x => formatBytes(x) + '/s'
+                      )
+                    : null}
+                  {evictTime && nursery.cells_tenured
+                    ? _markerDetail(
+                        'gctenurereatecells',
+                        'Tenure rate (cells/sec)',
```

Comment:
same question as above: do we need the "cells/sec" in the label ?

Reply:
Same as above.